cleanlab_studio/examples/python_api_demo.ipynb

```diff
@@ -0,0 +1,309 @@
+{
```

Comment:
since this is a tutorial / demo and not source code, we don't want to include this as part of the package. instead, we want to put this as part of our docs (see how the CLOS does it: https://github.com/cleanlab/cleanlab/tree/master/docs/source/tutorials)